Lib/test/test_pathlib.py

```diff
@@ -1643,6 +1643,33 @@ def test_rmdir(self):
         self.assertFileNotFound(p.stat)
         self.assertFileNotFound(p.unlink)
 
+    def test_rmtree(self):
+        p = self.cls(BASE) / 'dirB'
```

Comment:
Why do you use 2 folder p and q, and not only one?

Reply:
`p` refers to [dirB](https://github.com/asmith26/cpython/blob/fix-issue-33498/Lib/test/test_pathlib.py#L1214) which holds a file and a symlink.

`q` refers to [dirC](https://github.com/asmith26/cpython/blob/fix-issue-33498/Lib/test/test_pathlib.py#L1217) which holds sub-directories.

Thus I thought it would be useful to test as many scenarios as possible (I would be use the "[BASE](https://github.com/asmith26/cpython/blob/fix-issue-33498/Lib/test/test_pathlib.py#L1209)" directory, but it failed on [dirE](https://github.com/asmith26/cpython/blob/fix-issue-33498/Lib/test/test_pathlib.py#L1221) for permission issues).